pkg/controller/controller.go

```diff
@@ -522,3 +523,39 @@ func isInstanceReady(instance *v1alpha1.Instance) bool {
 
 	return false
 }
+
```

Comment:
Probably yes, there is already similar code in [deployment_util.go#L568](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/deployment/util/deployment_util.go#L568), [history.go#L276](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/history.go#L276) and many other places